yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -950,21 +953,13 @@ func fetchUnsupportedQueryConstructs() ([]utils.UnsupportedQueryConstruct, error
 	var result []utils.UnsupportedQueryConstruct
 	for i := 0; i < len(executedQueries); i++ {
 		query := executedQueries[i]
-
-		// Check if the query starts with CREATE, INSERT, UPDATE, or DELETE
-		upperQuery := strings.ToUpper(strings.TrimSpace(query))
-		if strings.HasPrefix(upperQuery, "CREATE") || strings.HasPrefix(upperQuery, "INSERT") ||
-			strings.HasPrefix(upperQuery, "UPDATE") || strings.HasPrefix(upperQuery, "DELETE") {
-			continue
-		}
-
+		log.Debugf("fetched unsupported query constructs for query-%s", query)
 		queryParser := queryparser.New(query)
 		err := queryParser.Parse()
 		if err != nil {
 			return nil, fmt.Errorf("failed to parse query-%s: %w", query, err)
```

Comment:
Just thinking: should we log the error here as well and not return, just in case there is some query in pgss that parser failed to parse? Not sure if there can be any case.

Reply:
Yeah, log.Error and continue and makes sense to me!